src/network/networkd-network.h

```diff
@@ -49,6 +49,15 @@ typedef enum IPv6PrivacyExtensions {
         _IPV6_PRIVACY_EXTENSIONS_INVALID = -1,
 } IPv6PrivacyExtensions;
 
+typedef enum IPv6AcceptRA {
+        IPV6_ACCEPT_RA_NO,
+        IPV6_ACCEPT_RA_YES,
+        IPV6_ACCEPT_RA_KERNEL,
+        IPV6_ACCEPT_RA_KERNEL_DISABLED,
```

Comment:
I cannot find any difference between `IPV6_ACCEPT_RA_YES` and `IPV6_ACCEPT_RA_KERNEL_DISABLED`. Maybe, we can drop `IPV6_ACCEPT_RA_KERNEL_DISABLED`?

Also, please add brief comments about each values. I guess
`IPV6_ACCEPT_RA_NO`: disable RA on networkd and kernel.
`IPV6_ACCEPT_RA_YES`: enable RA on networkd, and disable it on kernel.
`IPV6_ACCEPT_RA_KERNEL`: disable RA on networkd, and enable it on kernel.
`IPV6_ACCEPT_RA_KERNEL_DISABLED`: what???